### ISSUE TITLE
UI Bug

### ISSUE THREAD

**Issue by JimZiii** [LagradOst/CloudStream-3#577]
On some shows, for example Dragon ball Z on some providers, you can't reach the tab to browse later episodes. You can access everything above and under but not the episode tab where you switch episodes page.

At least with the remote on a nvidia shield pro 2019

**Comment by LagradOst** [comment]
Can you please provide a screenshot and highlight the elements that are inaccesable.

**Comment by JimZiii** [comment]
![20220206_013602](https://user-images.githubusercontent.com/7854537/152663815-1bc6b9fc-40f8-4418-b062-47c971570808.jpg)
Where it says 1-50.
Its not on all providers, i guess the layout changes slightly sometimes depending on the information they have available for each particular show

**Comment by akash-258** [comment]
Yes, I have the same issue. I can't watch anything on firestick due to this bug, only small anime which have few episodes. I was trying to watch Naruto from episodes 300+  but can't reach there.
